src/components/navigation/Navbar.tsx

```diff
@@ -1,12 +1,25 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import "./Navbar.scss";
 
 export const Navbar: React.FunctionComponent = () => {
   return <nav>
-    <ul>
-      <li><Link to="/">Home</Link></li>
-      <li><Link to="/rovers">Rovers</Link></li>
-      <li><Link to="/quiz">Quiz</Link></li>
-    </ul>
+
+    <div className="emptySpacingBox"></div>
+
+    <div id="menuToggle">
```

Comment:
Prefer classes to ids
```suggestion
    <div className="menuToggle">
```

Reply:
Done